backend/controllers/userController.js

```diff
@@ -0,0 +1,60 @@
+// set up database 
+const db = require('../dbModel.js');
+const bcrypt = require('bcrypt');
+const saltFactor = 10;
+const userController = {};
+
+userController.signUp = (req, res, next) => {
+  const { username, password } = req.body;
+  bcrypt.hash(password, saltFactor, (err, hash) => {
+    const signupQuery = 'INSERT INTO users (username, password, highscore) VALUES ($1, $2, 0)';
```

Comment:
What's the `highscore` field in this query for?

Reply:
oops I copied those few lines from another project haha forgot to remove it xD